openapi/paths/journal-entries@{id}.yaml

```
@@ -0,0 +1,64 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Journal
+  summary: Retrieve journal entry
+  operationId: GetJournalEntry
+  x-sdk-operation-name: getEntry
+  description: Retrieves a journal entry with a predefined identifier.
```

nitpick: based on our style
```suggestion
  description: Retrieves a journal entry with a specified ID.
```